clang/include/clang/Basic/TokenKinds.def

```diff
@@ -660,6 +660,9 @@ KEYWORD(out                         , KEYHLSL)
 #define HLSL_INTANGIBLE_TYPE(Name, Id, SingletonId) KEYWORD(Name, KEYHLSL)
 #include "clang/Basic/HLSLIntangibleTypes.def"
 
+// HLSL Type traits
+TYPE_TRAIT_1(__builtin_is_intangible, IsIntangibleType, KEYHLSL)
```

Comment:
Should this be named `__builtin_hlsl_is_intangible` so it's clear this is HLSL-specific?

Reply:
Sure, will do!